src/main/java/com/mraof/minestuck/world/gen/feature/RandomRockConditionFreeBlobFeature.java

```diff
@@ -0,0 +1,73 @@
+package com.mraof.minestuck.world.gen.feature;
+
+import com.mojang.datafixers.Dynamic;
+import com.mraof.minestuck.block.MSBlocks;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.gen.ChunkGenerator;
+import net.minecraft.world.gen.GenerationSettings;
+import net.minecraft.world.gen.feature.Feature;
+
+import java.util.Iterator;
+import java.util.Random;
+import java.util.function.Function;
+
+public class RandomRockConditionFreeBlobFeature extends Feature<RandomRockBlockBlobConfig>
```

Comment:
If you would like to make changes, you are welcome to. But because I cannot form a clear opinion in my head due to not having the same comprehension level as you, I will keep it the same for now